### ISSUE TITLE
Test the gating support by feeding it into an MLP

### ISSUE THREAD

**Issue by ruaruasss** [ruaruasss/Thesis_code#1]
I did it in model3.py

**Comment by walidgeuttala** [comment]
if you did it, report the results, and close the issue

**Comment by ruaruasss** [comment]
### **original one gets the best results, but sum is better than concat.**

testam(10308290)
Average Training Time: 113.3158 secs/epoch
Average Inference Time: 4.6319 secs
The valid loss on best model is 2.8994
On average over 12 horizons, Test MAE: 3.1811, Test MAPE: 0.0848, Test RMSE: 6.1464

concat(torch.cat(g1,g2,g3)): (10498629)
Average Training Time: 113.4038 secs/epoch
Average Inference Time: 4.5701 secs
The valid loss on best model is 3.0598
On average over 12 horizons, Test MAE: 3.3841, Test MAPE: 0.0937, Test RMSE: 6.8526

sum(g1+g2+g3): (10498771)
Average Training Time: 113.0891 secs/epoch
Average Inference Time: 4.5854 secs
The valid loss on best model is 2.9069
On average over 12 horizons, Test MAE: 3.2317, Test MAPE: 0.0898, Test RMSE: 6.6169
Total time spent: 4742.5133
